### PR TITLE
Table updates, Scraper Addition

### PR DESCRIPTION
**Pull request by EricaPomme**
As noted in the committed changelog, this brings a few more updates. Namely, the drink table has been recreated with values from Wowhead as of 2023/06/04, and a new directory has been added with a tool to easily create properly formatted lines. It's a bit of a quick and dirty tool; I probably could have spent the time I spent writing out the readme making it an all-in-one does-everything table generator but I never claimed to be clever. :)
CleanShot 2023-06-04 at 09 01 46@2x

### CODE REVIEW

File: DrinkMacroCreator.lua [hunk 1]
```diff
@@ -14,184 +14,241 @@ Drinks Table (haha) Notes:
...
+    49254, -- Tarp Collected Dew ........................... (180 MP) [10]
+    60269 -- Well Water ................................... (180 MP) [10]
 }
 
+-- Merge the two tables
+table.move(drinks, 1, #drinks, #magefood + 1, magefood)
+for i = 1, #magefood do
+    drinks[i] = magefood[i]
+end
+
```
Comment (DavidPHH) [inline DrinkMacroCreator.lua]: WoW's lua doesn't have table.move as far as i can see? Could you change this to something else please.

```suggestion
for i = #magefood, 1, -1 do
    table.insert(drinks, 1, magefood[i])
end
```

**Comment by DavidPHH** [comment]
Very nice addition with the Scrapper, deffinitely beats doing it manually.

A possible way to speed it up could maybe be by appending &xml to the link, e.g. `https://www.wowhead.com/item=194684&xml` to avoid fetching the entire page. And also maybe doing the whole thing within python instead of half manually with bash & python.  

Although no point in spending more time on it right now IMO, since new drinks are added so infrequently. This seems plenty good enough.

Thanks for the contributions 👍

**Review by DavidPHH — CHANGES_REQUESTED** [review]

**Comment by EricaPomme** [comment]
I tried the xml thing initially, it looked like it didn’t have all the information we needed in the page that was returned. 

This was one of those “banged it out for giggles while watching YouTube on the couch” things, so I’ll admit some laziness in the approach. I’ll take another swing at the scraper; the more I think about it, the more I’d rather just have a “push button, receive table” tool. 

Edit: Disregard that. I'm an idiot. XML is just fine. Rekerjiggering this as I should have done in the first place.

**Comment by EricaPomme** [comment]
I gutted it and rebuilt it like I knew I should have in the first place. Sorry for wasting your time with the first PR. 😅

**Review by DavidPHH — APPROVED** [review]
Thank you!
LGTM

**Pull request merged**